src/scss/components/navigation/_language-switcher.scss

```diff
@@ -0,0 +1,105 @@
+.language-switcher {
+  position: relative;
+
+  .language-link {
+    @include font-size(16px);
+    @include font-weight($font-weight-regular);
```

Comment:
Is this wrapper required? I think just the variable does the same thing. I don't remember if I have asked this before but I will ask it again :D

Reply:
Let's remove the mixin so we won't use it again.